models/analytics.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+type EmbeddingId int
```

Comment:
I'd call it "type" rather than "id" because "id" implies "unique identifier" to me, but just a matter of personal taste perhaps